package.json

```diff
@@ -43,9 +43,11 @@
     "chai": "^4.2.0",
     "commander": "^2.19.0",
     "core-js": "^2.6.3",
+    "delay": "^4.4.0",
     "eslint": "^7.10.0",
     "eslint-config-digitalbazaar": "^2.6.1",
     "mocha": "^5.2.0",
+    "nsolid": "0.0.0",
```

Comment:
What's the story with this package?  NPM page is strange as well: https://www.npmjs.com/package/nsolid

Reply:
The nsolid executable does something kind of wizardry to replace what's in it/what it runs when you use it instead of `node` directly. So this is "proper" despite being really weird.